npm/src/index.d.ts

```diff
@@ -1,21 +1,23 @@
-type inputType = 'lli' | 'qasm';

```

Comment:
Looks like there were `\r` line endings that were auto removed,have to hide white space to view